powerspikegg/go_gateway/gateway.go

```diff
@@ -14,9 +14,12 @@ import (
 )
 
 var (
-	grpcServerAddress = flag.String("grpc-server-address", "127.0.0.1:50001", "Address of the GRPC server")
-	httpAddress       = flag.String("http-address", "127.0.0.1", "Address of the HTTP gateway")
-	httpPort          = flag.String("http-port", "8080", "Port of the HTTP gateway")
+	grpcFetcherAddress     = flag.String("grpc_fetcher_address", "127.0.0.1", "Address of the GRPC server of the fetcher service")
```

Comment:
For later: Since all our stack in backend uses grpc, you may avoid specifying `grpc_*` in your flags.